lang/main.json

```diff
@@ -462,6 +462,9 @@
         "busyTitle": "All recorders are currently busy",
         "buttonTooltip": "Start / Stop recording",
         "error": "Recording failed. Please try again.",
+        "expandedOff": "Recording has stopped",
+        "expandedOn": "The meeting is currently being recorded.",
+        "expandedPending": "Recording is being started...",
```

Comment:
s/pending/inProgress/ ?

Reply:
I wanted the naming to match the name of the status itself, see:

```
switch (_status) {
        case JitsiRecordingConstants.status.OFF:
            postfix = 'expandedOff';
            break;
        case JitsiRecordingConstants.status.PENDING:
            postfix = 'expandedPending';
            break;
        case JitsiRecordingConstants.status.ON:
            postfix = 'expandedOn';
            break;
        }
```